Makefile

```diff
@@ -129,6 +129,8 @@ wheel:
 
 	$(PYTHON) $(MK_DIR)/setup.py bdist_wheel
 
+	rm -r $(MK_DIR)/build
+
```

Comment:
Why did you add this here instead of `clean` and `clean-all`?

Reply:
This build folder is only used temporarily when building the wheel, it is not itself the output of the wheel command. I don't think we need to keep it around once the wheel artifact is produced (it just pollutes the user directory).